pkg/asset/cluster/tfvar.go

```diff
@@ -0,0 +1,83 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/types/config"
+)
+
+const (
+	tfvarFilename  = "terraform.tfvar"
+	tfvarAssetName = "Terraform tfvar file"
```

Comment:
@crawford seems to prefer inline title case for these:

```console
$ git grep -A1 ' Name(' origin/master -- pkg/asset/ | head -n11
origin/master:pkg/asset/ignition/bootstrap.go:func (a *bootstrap) Name() string {
origin/master:pkg/asset/ignition/bootstrap.go-	return "Bootstrap Ignition Config"
--
origin/master:pkg/asset/ignition/master.go:func (a *master) Name() string {
origin/master:pkg/asset/ignition/master.go-	return "Master Ignition Config(s)"
--
origin/master:pkg/asset/ignition/testasset_test.go:func (a *testAsset) Name() string {
origin/master:pkg/asset/ignition/testasset_test.go-	return "Test Asset"
--
origin/master:pkg/asset/ignition/worker.go:func (a *worker) Name() string {
origin/master:pkg/asset/ignition/worker.go-	return "Worker Ignition Config"
```

Can we follow that pattern with:

```go
// Name returns the human-friendly name of the asset.
func (t *TerraformVariables) Name() string {
  return "Terraform Variables"
}
```

Reply:
Updated